slo_generator/backends/cloud_service_monitoring.py

```diff
@@ -342,7 +356,8 @@ def create_slo(self, window, slo_config):
         return SSM.to_json(slo)
 
     @staticmethod
-    def build_slo(window, slo_config):  # pylint: disable=R0912,R0915
+    def build_slo(window: int, # pylint: disable=R0912,R0915
```

Comment:
I just noticed that the `delete()` function above (line 122) has a similar multi-line signature with a Pylint annotation. It is probably cleaner to put the annotation on a separate line, just above the function name (assuming it still works as expected).

Apologies for the back and forth on this one, but I find it cleaner this way.

Reply:
Yeah much cleaner. No worries I'll fix that